src/wallet/spend.cpp

```diff
@@ -1128,6 +1225,36 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
         available_coins = AvailableCoins(wallet, &coin_control, coin_selection_params.m_effective_feerate);
     }
 
+    // Load a json file that describes a target utxo set
```

Comment:
Loading this file every time we fund a transaction doesn't seem reasonable. I think we should start with a static file that needs a restart whenever the node operator wants to change values, and we can later decide how to make this more dynamic (if it's even necessary).

Reply:
That makes sense. I also think a static file could describe the buckets in a way that better adapts to a changing fee environment.

For example, rather than `start_satoshis` and `end_satoshi` we could define a bucket as:

```json
        "confirmation_target": 6,
        "fee_rate_std": 10000, 
        "buckets": [
        {
            "target_satoshis": 10000,
            "target_utxo_count": 150
        },
```

The range of values you want to spend should be "target_satoshis" + the fee to spend that input at a fee rate that will confirm in "confirmation_target" blocks. Because free rates vary, we randomize the current fee rate within a range +/- "fee_rate_std".

To refill a bucket, compute the `target_output` as:
```
target_feerate = current_feerate("confirmation_target") + random(-1 * "fee_rate_std", "fee_rate_std")
target_output = "target_satoshis" + size_of_input * target_feerate 
``` 
Ideally we would only need to restart when we add/remove buckets, change their counts or when fee variance changes dramatically.